app/Phragile/PhabricatorAPI.php

```diff
@@ -116,4 +116,23 @@ public function queryPHIDs(array $phids)
 			['phids' => $phids]
 		);
 	}
+
+	public function getSprintDuration($phid)
+	{
+		try
+		{
+			$duration = $this->client->callMethodSynchronous(
+				'sprint.getstartenddates',
+				['project' => $phid]
+			);
```

Comment:
Yup, it'll use this if it's installed, otherwise it'll let you enter the dates manually.